packages/fluent-theme/src/fluent/styles/Toggle.styles.ts

```diff
@@ -1,14 +1,46 @@
-export const ToggleStyles = {
-  pill: {
-    width: '40px',
-    height: '20px',
-    borderRadius: '10px',
-    padding: '0 4px'
-  },
-  thumb: {
-    width: '12px',
-    height: '12px',
-    borderRadius: '12px',
-    borderColor: 'transparent'
-  }
+import { IToggleProps } from '../../../../office-ui-fabric-react/lib';
+import { CommunicationColors, NeutralColors } from '../FluentColors';
+
+export const ToggleStyles = (props: IToggleProps) => {
+  const { disabled, checked } = props;
+  return {
+    pill: [
+      {
+        width: '40px',
+        height: '20px',
+        borderRadius: '10px',
+        padding: '0 4px'
+      },
+      !disabled && [
+        checked && {
+          selectors: {
+            ':hover': [
+              {
+                backgroundColor: CommunicationColors.shade20
```

Comment:
The `palette` object should be used wherever possible. See if this color and the two below exist in that object before using the Fluent palettes.